README.md

```diff
@@ -41,8 +41,8 @@ Before starting your exercises, be sure to have your environment fully working b
 - Introduction to Bad Flask App ([02-bad-flask-app-intro.md](exercises/02-bad-flask-app.md))
 - Introduction to XPaths ([03-xpath-intro.md](exercises/03-xpath-intro.md))
 - Handling iframes ([04-iframes.md](exercises/04-iframes.md))
-- More difficult XPaths ([05-filling-fields.md](exercises/05-filling-fields.md))
-- Extend your libraries with Python ([06-datepicker.md](exercises/06-datepicker.md))
+- Using complex XPaths ([05-filling-fields.md](exercises/05-filling-fields.md))
+- Extending your libraries with Python ([06-datepicker.md](exercises/06-datepicker.md))
```

Comment:
This isn't actually true anymore. We're not creating a new library anymore, but rather be more aware of already existing libraries.